src/makeCalculator.js

```diff
@@ -5,6 +5,29 @@
  */
 function makeCalculator() {
   // write code here
+  let result = 0;
+
+  const calculator = {
+    add: (num) => result + num,
+    subtract: (num) => result - num,
+    divide: (num) => result / num,
+    multiply: (num) => result * num,
```

Comment:
The methods `add`, `subtract`, `multiply`, `divide` should update the `result` property. Currently, they calculate the value but do not update `result`. You need to assign the new value to `result` using the `=` operator.